dev-python/numpy/numpy-1.11.0.ebuild

```diff
@@ -0,0 +1,151 @@
+HOMEPAGE="http://www.numpy.org/"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz
+	doc? (
+		http://docs.scipy.org/doc/${DOC_P}/${PN}-html-${DOC_PV}.zip
+		http://docs.scipy.org/doc/${DOC_P}/${PN}-ref-${DOC_PV}.pdf
+		http://docs.scipy.org/doc/${DOC_P}/${PN}-user-${DOC_PV}.pdf
+	)"
+# It appears the docs haven't been upgraded, still @ 1.8.1
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~mips ~ppc ~ppc64 ~s390 ~sh ~sparc ~x86 ~amd64-fbsd ~x86-fbsd ~x86-freebsd ~x86-interix ~amd64-linux ~arm-linux ~x86-linux ~ppc-macos ~x64-macos ~x86-macos ~sparc-solaris ~x64-solaris ~x86-solaris"
+IUSE="doc lapack test"
+
+RDEPEND="
+	dev-python/setuptools[${PYTHON_USEDEP}]
+	lapack? ( virtual/cblas virtual/lapack )"
+DEPEND="${RDEPEND}
+	doc? ( app-arch/unzip )
+	lapack? ( virtual/pkgconfig )
+	test? ( >=dev-python/nose-1.0[${PYTHON_USEDEP}] )"
+
+PATCHES=(
+	"${FILESDIR}"/${PN}-1.10.2-no-hardcode-blas.patch
+)
+
+src_unpack() {
+	default
+	if use doc; then
+		unzip -qo "${DISTDIR}"/${PN}-html-${DOC_PV}.zip -d html || die
+	fi
+}
+
+pc_incdir() {
+	$(tc-getPKG_CONFIG) --cflags-only-I $@ | \
+		sed -e 's/^-I//' -e 's/[ ]*-I/:/g' -e 's/[ ]*$//' -e 's|^:||'
+}
+
+pc_libdir() {
+	$(tc-getPKG_CONFIG) --libs-only-L $@ | \
+		sed -e 's/^-L//' -e 's/[ ]*-L/:/g' -e 's/[ ]*$//' -e 's|^:||'
+}
+
+pc_libs() {
+	$(tc-getPKG_CONFIG) --libs-only-l $@ | \
+		sed -e 's/[ ]-l*\(pthread\|m\)\([ ]\|$\)//g' \
+		-e 's/^-l//' -e 's/[ ]*-l/,/g' -e 's/[ ]*$//' \
+		| tr ',' '\n' | sort -u | tr '\n' ',' | sed -e 's|,$||'
+}
+
+python_prepare_all() {
+	if use lapack; then
+		append-ldflags "$($(tc-getPKG_CONFIG) --libs-only-other cblas lapack)"
+		local libdir="${EPREFIX}"/usr/$(get_libdir)
+		cat >> site.cfg <<-EOF
+			[blas]
+			include_dirs = $(pc_incdir cblas)
+			library_dirs = $(pc_libdir cblas blas):${libdir}
+			blas_libs = $(pc_libs cblas blas)
+			[lapack]
+			library_dirs = $(pc_libdir lapack):${libdir}
+			lapack_libs = $(pc_libs lapack)
+		EOF
+	else
+		export {ATLAS,PTATLAS,BLAS,LAPACK,MKL}=None
+	fi
+
+	export CC="$(tc-getCC) ${CFLAGS}"
+
+	append-flags -fno-strict-aliasing
+
+	# See progress in http://projects.scipy.org/scipy/numpy/ticket/573
+	# with the subtle difference that we don't want to break Darwin where
+	# -shared is not a valid linker argument
+	if [[ ${CHOST} != *-darwin* ]]; then
+		append-ldflags -shared
+	fi
+
+	# only one fortran to link with:
+	# linking with cblas and lapack library will force
+	# autodetecting and linking to all available fortran compilers
+	append-fflags -fPIC
+	if use lapack; then
+		NUMPY_FCONFIG="config_fc --noopt --noarch"
+		# workaround bug 335908
+		[[ $(tc-getFC) == *gfortran* ]] && NUMPY_FCONFIG+=" --fcompiler=gnu95"
+	fi
+
+	# don't version f2py, we will handle it.
+	sed -i -e '/f2py_exe/s: + os\.path.*$::' numpy/f2py/setup.py || die
+
+	# we don't have f2py-3.3
+	sed \
+		-e 's:test_f2py:_&:g' \
+		-i numpy/tests/test_scripts.py || die
+
+	distutils-r1_python_prepare_all
+	eapply_user
```

Comment:
`distutils-r1_python_prepare_all` already calls `default` in `EAPI=6`, hence a separate `eapply_user` shouldn't be necessary.